control-plane-operator/controllers/hostedcontrolplane/scheduler/deployment.go

```diff
@@ -86,7 +90,7 @@ func buildSchedulerContainerMain(image, namespace string, featureGates []string,
 		c.Args = []string{
 			fmt.Sprintf("--config=%s", configPath),
 			fmt.Sprintf("--cert-dir=%s", certWorkDir),
-			"--port=0",
+			fmt.Sprintf("--secure-port=%d", schedulerSecurePort),
```

Comment:
Do we need to keep the old arg to disable the insecure port in versions < 4.10/1.23?

Reply:
Upstream removed that flag in 1.23

Reply:
https://github.com/kubernetes/kubernetes/pull/96345

Reply:
yeah, I was getting a complaint in the log about it, so I decided to get rid of it

Reply:
Ok, lets assume that this isn't a security issue for previous versions